plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.scss

```
@@ -45,6 +45,18 @@
 		text-align: center;
 		z-index: 26;
 	}
+
+	&__update-count-extensions {
+		background-color: #000;
+	}
+
+	&__update-count-themes {
+		background-color: #000;
+	}
+
+	&__update-count-business-services {
+		background-color: #000;
+	}
```

How about this or just using the same class name for all those elements?
```suggestion
	&__update-count-extensions,
	&__update-count-themes,
	&__update-count-business-services {
		background-color: #000;
	}
```

---

I see that there's a class `woocommerce-marketplace__update-count` already so it could be just

```css
&__update-count {
	background-color: #000;
}
```

---

Also, based on the designs I see that only active tab should have dark bg. Inactive tabs should have darker font and lighter background.

---

> Also, based on the designs I see that only active tab should have dark bg. Inactive tabs should have darker font and lighter background.

This is dependent on the behavior you noticed (search cleared when clicking on a tab). I will note it down and address it in that other issue.

---

CSS styling unified in ca443b7fda

---

@mcliwanow just FYI I will revert https://github.com/woocommerce/woocommerce/commit/ca443b7fdac9e99c3b5d4c67fe63db4eb54ea297 in an upcoming PR and use your proposed suggestion. I forgot that I already tried it and it also changes the `My subscriptions` tab, which I assume we want to keep red.

```
	&__update-count-extensions,
	&__update-count-themes,
	&__update-count-business-services {
		background-color: #000;
	}
```